tsl/src/bgw_policy/policy_utils.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+
+#include <postgres.h>
+#include <utils/builtins.h>
+#include "dimension.h"
+#include "jsonb_utils.h"
+#include "policy_utils.h"
+
+/* Helper function to compare jsonb label value
+ * and passed in value.
+ * This function is used for labels
+ * defined on the hypertable's dimension
+ * Parameters:
+ * config - jsonb config value
+ * label - label we are looking for inside the config
```

Comment:
I have been using this style for comments elsewhere, which is easier on the eye. Also matches the way you format the `Returns:` below.
```suggestion
 * Parameters:
 *   - config: jsonb config value
 *   - label: label we are looking for inside the config
```